src/components/sidebar/components/sidebar-nav/sidebar-nav-menu-item/index.tsx

```diff
@@ -107,9 +108,13 @@ const SidebarNavSubmenu: React.FC<SidebarMenuItemProps> = ({ item }) => {
   )
 }
 
+export function HorizontalRule({ className }: { className?: string }) {
+  return <hr className={classNames(s.divider, className)} />
+}
```

Comment:
Can you remove the `export` from here and move `HorizontalRule` down to the `export { SidebarNavLink }` statement? It makes it easier to tell what all is exported from this file and what isn't.

Reply:
✅ Great call, moved!